internal/js/modules/k6/browser/tests/get_by_test.go

```diff
@@ -0,0 +1,770 @@
+// practically none of this work on windows
+//go:build !windows
+
+package tests
+
+import (
+	"testing"
+
+	"github.com/grafana/sobek"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.k6.io/k6/internal/js/modules/k6/browser/common"
+)
+
+func ptr[T any](v T) *T {
+	return &v
+}
```

Comment:
For consistency, I'd move `ptr` next to [these functions](https://github.com/grafana/k6/blob/113abf5fdd7bc5ddc36c87a10aa7febdf7a0bb71/internal/js/modules/k6/browser/tests/test_browser.go#L352-L380). Instead of `ptr`, `toPtr` or `ptrOf` could be more readable and understandable about what it does, but I don't have a weak opinion.